src/VirtoCommerce.CatalogModule.Data/Repositories/CatalogDbContext.cs

```diff
@@ -113,9 +113,50 @@ protected override void OnModelCreating(ModelBuilder modelBuilder)
                 .OnDelete(DeleteBehavior.Restrict);
             modelBuilder.Entity<PropertyEntity>().HasOne(m => m.Category).WithMany(x => x.Properties).HasForeignKey(x => x.CategoryId)
                 .OnDelete(DeleteBehavior.Restrict);
+            modelBuilder.Entity<PropertyEntity>().HasOne(m => m.PropertyGroup).WithMany().HasForeignKey(x => x.PropertyGroupId)
+                .OnDelete(DeleteBehavior.Restrict);
 
             #endregion Property
 
+            #region PropertyGroup
+
+            modelBuilder.Entity<PropertyGroupEntity>().ToTable("PropertyGroup").HasKey(x => x.Id);
+            modelBuilder.Entity<PropertyGroupEntity>().Property(x => x.Id).HasMaxLength(128).ValueGeneratedOnAdd();
+            modelBuilder.Entity<PropertyGroupEntity>().HasOne(m => m.Catalog).WithMany(x => x.PropertyGroups).HasForeignKey(x => x.CatalogId)
+                .OnDelete(DeleteBehavior.Restrict);

```

Comment:
Use `IdLength` instead of `128`
Use `bulder => { ... }` for consistency with localized entities